Content/css/style_index.css

```diff
@@ -0,0 +1,79 @@
+* {
```

Comment:
According to the sir commit message should be in this format `FT2023-61: short description.`